src/SDK/SmartDetectors.Extensions/SmartDetectors.Extensions.csproj

```diff
@@ -9,7 +9,7 @@
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
   </PropertyGroup>  
   <ItemGroup>
-    <PackageReference Include="Microsoft.Azure.Monitoring.SmartDetectors.RuntimeEnvironment.Contracts" Version="0.16.0" />
+    <PackageReference Include="Microsoft.Azure.Monitoring.SmartDetectors.RuntimeEnvironment.Contracts" Version="1.0.9.177" />
```

Comment:
>1.0.9.177 [](start = 112, length = 9)

don't you have to update Nuget.config?

Reply:
I updated

---
In reply to: [257946149](https://github.com/Azure/SmartDetectorsMonitoringAppliance/pull/49#discussion_r257946149) [](ancestors = 257946149)

Reply:
yes - add the following line to nuget.config:
<add key="OneBranchOfficial" value="[https://msazure.pkgs.visualstudio.com/DefaultCollection/_apis/packaging/Official/nuget/index.json](https://msazure.pkgs.visualstudio.com/DefaultCollection/_apis/packaging/Official/nuget/index.json)" />

---
In reply to: [258010304](https://github.com/Azure/SmartDetectorsMonitoringAppliance/pull/49#discussion_r258010304) [](ancestors = 258010304,257946149)